src/zod/openapi.md

```
@@ -4,72 +4,107 @@ label: OpenAPI
 
+# ZOD Version 4 Support
+
+Danet now supports Zod version 4 `@danet/zod@^0.1.0`, which is a major upgrade from version 3.
+
+::: note
+All previous versions of `@danet/zod` support only Zod version 3.
+:::
+
+One of the new features is the ability to generate JSON Schema from Zod schemas and vice versa.
+
+### z.fromJSONSchema()
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)